Projects/Foundations/Schemas/Docs-Topics/T/Tested/Tested-Exchanges/tested-exchanges-003-binancefutures-test-status.json

```diff
@@ -1,114 +0,0 @@
-{
-    "topic": "Tested Exchanges",
-    "pageNumber": 3,
-    "type": "Binance Futures USDS-M/COIN-M Test Status",
-    "definition": {
-        "text": "Binance Futures Perpetual contracts are partially tested for data mining. Testing in progress for live trading.  Note that Quarterly Contract trading will be tested in the future based on users requirement/demand.",
-        "updated": 1615141847424,
-        "translations": [
-            {
-                "language": "RU",
-                "text": "Бессрочные контракты Binance Futures частично протестированы для добычи данных. Тестирование для реальной торговли продолжается. Обратите внимание, что торговля квартальными контрактами будет протестирована в будущем на основе требований/спроса пользователей.",
-                "updated": 1646900423397
-            }
-        ]
-    },
-    "paragraphs": [
-        {

```

Comment:
Do you want to delete this file?

Reply:
Yes please. I've a copy of that file and I will push it again once orders of the pages are sorted